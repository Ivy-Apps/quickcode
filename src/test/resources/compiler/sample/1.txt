#if {{isViewModel}} #then
@get:Rule
val rule = CompositionViewModelTestRule()
#endif
#if {{coroutineTest}} #then
@get:Rule
val coroutineTestRule = CoroutineTestRule()
#endif

#if {{isViewModel}} #then
private lateinit var viewModel: {{className}}
#else
private lateinit var sut: {{className}}
#endif

@Before
fun setup() {
#if {{isViewModel}} #then
    viewModel = {{className}}(
    )
#else
    sut = {{className}}(
    )
    sdfljsdf
    sdjflsdj
    fjsdl

#endif
}