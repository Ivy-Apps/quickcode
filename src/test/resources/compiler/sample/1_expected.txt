@get:Rule
val rule = CompositionViewModelTestRule()

private lateinit var viewModel: MyClass

@Before
fun setup() {
    viewModel = MyClass(
    )
}